gnomad_constraint/utils/constraint.py

```diff
@@ -473,7 +473,11 @@ def apply_models(
         hl.case()
         .when(ht.coverage == 0, 0)
         .when(ht.coverage >= high_cov_definition, 1)
-        .default(coverage_model[1] * hl.log10(ht.coverage) + coverage_model[0])
+        .default(
+            (coverage_model[1] * hl.log10(ht.coverage) + coverage_model[0])
+            if coverage_model is not None
+            else 1
+        )
```

Comment:
Are globals OK with None now? I think they are, but I know this is an issue I have hade in the past:
```    # Annotate global annotations.
    ht = ht.annotate_globals(
        apply_model_params=hl.struct(
            max_af=max_af,
            pops=pops,
            plateau_models=plateau_models,
            coverage_model=coverage_model,
        )
    )
```

Reply:
looks like it's still an issue